package.json

```
@@ -7,7 +7,7 @@
         "server-install": "cd ./antalmanac-backend && if test -f package.json; then npm install; fi",
         "dependency-install": "concurrently -n \"BACKEND,FRONTEND\" \"npm run server-install\" \"npm run client-install\"",
         "postinstall": "npm run dependency-install",
-        "server": "cd antalmanac-backend && sls offline --stage development --noPrependStageInUrl",
+        "server": "cd ./antalmanac-backend && npm start",
```

```suggestion
        "server": "npm run start --prefix ./antalmanac-backend",
```

For consistently, let's use `--prefix` like we do with the frontend

---

Added, makes sense